src/content/manifest/okteto-variables.mdx

```diff
@@ -43,6 +64,127 @@ To delete an existing variable, click on the **Delete** button on the right. You
   />
 </p>
 
+## Configuring dynamic endpoints
+
+You can create dynamic endpoints for external resources within your Okteto environment using `$OKTETO_ENV`. You can define `$variables`, assign them to an `OKTETO_ENV_VAR`, and make those ENV VARs available in other sections (e.g. `deploy`, `destroy`) of the Oketo manifest to generate dynamic endpoints for external resources. 
+
+The example below demonstrates adding variables to `$OKTETO_ENV` in the `deploy` section and then access those variables in a sample `destroy` section:
+
+```yaml
+deploy:
+  hashicorp/terraform:1.4
+  commands:
+  - name: Create AWS infrastructure
+    command: |
+      set -e
+      resourceName="${OKTETO_NAMESPACE}-oktacoshop"
+      region=$AWS_REGION
+
+      terraform apply -input=false -var "bucket_name=$resourceName" -var "queue_name=$resourceName" -var "region=$region" -auto-approve
+      s3Dashboard="https://s3.console.aws.amazon.com/s3/buckets/${resourceName}"
+
+      # ... shortened for brevity
+    {
+      echo "OKTETO_EXTERNAL_S3_ENDPOINTS_BUCKET_URL=$s3Dashboard"
+      echo "S3_BUCKET_NAME=$resourceName"
+    } >> "$OKTETO_ENV"
+
+destroy:
+  image: hashicorp/terraform:1.4
+  commands:
+  - name: Delete the AWS infrastructure
+    command: |
+      set -e
+      resourceName="${OKTETO_NAMESPACE}-oktacoshop"
+      region=$AWS_REGION
+      
+      # omitted for brevity 
+
+      terraform apply -input=false -var "bucket_name=$resourceName" -var "queue_name=$resourceName" -var "region=$region" -auto-approve --destroy
+```
+
+### **Default Environment Variables**
+
+These are automatically populated and can be referenced in your Okteto manifest’s `deploy`/`destroy` commands:
```

Comment:
@pchico83 I had suggested we remove the word "commands" here to avoid confusion since I understood your previous comment to mean variables can be used anywhere in the manifest YAML and not solely, specifically within the `commands:` config

Reply:
There is a slightly difference. All variables in the okteto manifest are expanded. But in th deploy/destroy commands they are also injected in the execution runtime. That means, my deploy section could by:
```
deploy:
   - make deploy
```
And I have a `Makefile` where `make deploy` is defined to access okteto variables and that would work.
In other words, `make deploy` doesn't have references to these variables, but its internal implementation does, and that works because we inject all these variables in the runtime environment of the deploy/destroy commands

Reply:
What about this?

> These are automatically expanded and can be referenced in your Okteto manifest’s `deploy`/`destroy` sections either directly (within the manifest) or in external files (e.g. a `Makefile`):